apis/Google.Cloud.Spanner.V1/Google.Cloud.Spanner.V1/SpannerDate.cs

```diff
@@ -0,0 +1,279 @@
+﻿// Copyright 2022 Google LLC
+// 
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// 
+//     https://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+using Google.Api.Gax;
+using System;
+using System.Globalization;
+
+namespace Google.Cloud.Spanner.V1
+{
+    /// <summary>
+    /// The SpannerDate type represents a logical calendar date, independent of time zone.
+    /// Represents dates with values ranging from January 1, 0001 A.D through December 31, 9999 A.D. in the Gregorian calendar.
+    /// </summary>
+    public readonly struct SpannerDate : IEquatable<SpannerDate>, IComparable<SpannerDate>, IComparable
+    {
+        // TODO : Once upgraded to .NET 6, have an operator for converting DateOnly to SpannerDate and vice versa.
+
+        // The latest possible date is 9999/12/31 (DateTime.MaxValue) and the earliest possible date is 0001/01/01 (DateTime.MinValue).
+        // The days between maximum and minimum date is MaxNumberOfDays.
+        private const int MaxNumberOfDays = 3652058;
+
+        /// <summary>
+        /// Gets the latest possible date that can be created.
```

Comment:
"created" => "represented by this type" perhaps?

Reply:
Done.